invenio_jsonschemas/views.py

```diff
@@ -46,8 +54,21 @@ def create_blueprint(state):
     def get_schema(schema_path):
         """Retrieve a schema."""
         try:
-            return send_from_directory(state.get_schema_dir(schema_path),
-                                       schema_path)
+            if current_app.config['JSONSCHEMAS_REPLACE_REFS']:
```

Comment:
It should probably check for a `request` argument (e.g. `?refs=resolve`, `expand=1`, or similar).